### PR TITLE
Improving documentation of `models.encodings.entangling_layer`

### PR DESCRIPTION
**Pull request by renatomello**
Checklist:
- [x] Reviewers confirm new code works as expected.
- [x] Tests are passing.
- [x] Coverage does not decrease.
- [x] Documentation is updated.

### CODE REVIEW

File: doc/source/api-reference/qibo.rst [hunk 1]
```diff
@@ -390,6 +390,26 @@ Entangling layer
...
 
 Generates a layer of nearest-neighbour two-qubit gates, assuming 1-dimensional connectivity.
+With the exception of :class:`qibo.gates.gates.GeneralizedfSim`,
+can be set to any of the two-qubit gates implemented in ``qibo``.
+If the chosen gate is parametrized, all phases are set to :math:`0.0`.
+Note that these phases can be updated a posterior by using
+:meth:`qibo.models.Circuit.set_parameters`.
+There are four possible choices of layer ``architechture``:
+``diagonal``, ``shifted``, ``even-layer``, and ``odd-layer``.
+For instance, we show below an example for each architecture and ``nqubits = 6``.
```
Comment (renatomello) [inline doc/source/api-reference/qibo.rst:400]:
```suggestion
For instance, we show below an example of each architecture for ``nqubits = 6``.
```

**Review by Simone-Bordoni — APPROVED** [review]

File: doc/source/api-reference/qibo.rst [hunk 2]
```diff
@@ -390,6 +390,26 @@ Entangling layer
 """"""""""""""""
 
 Generates a layer of nearest-neighbour two-qubit gates, assuming 1-dimensional connectivity.
+With the exception of :class:`qibo.gates.gates.GeneralizedfSim`,
+can be set to any of the two-qubit gates implemented in ``qibo``.
+If the chosen gate is parametrized, all phases are set to :math:`0.0`.
+Note that these phases can be updated a posterior by using
+:meth:`qibo.models.Circuit.set_parameters`.
+There are four possible choices of layer ``architechture``:
```
Comment (MatteoRobbiati) [inline doc/source/api-reference/qibo.rst:398]:
```suggestion
There are four possible choices of layer ``architecture``:
```

File: doc/source/api-reference/qibo.rst [hunk 3]
```diff
@@ -390,6 +390,26 @@ Entangling layer
 """"""""""""""""
 
 Generates a layer of nearest-neighbour two-qubit gates, assuming 1-dimensional connectivity.
+With the exception of :class:`qibo.gates.gates.GeneralizedfSim`,
+can be set to any of the two-qubit gates implemented in ``qibo``.
```
Comment (MatteoRobbiati) [inline doc/source/api-reference/qibo.rst:394]:
```suggestion
any of the two-qubit gates implemented in ``qibo`` can be selected to customize the entangling layer.
```

**Review by MatteoRobbiati — CHANGES_REQUESTED** [review]
Thanks @renatomello!

I think it would be better to standardize a bit more the images in the section. To do this, we should keep the width in the usual range of `400px-800px`, and regulate the height. To do so, I opened #1278, where the `scale` tag is removed and the images widths and heights are rescaled to fit the aforementioned range.

A couple of suggestions follow.

**Review by MatteoRobbiati — APPROVED** [review]
Thanks.

**Pull request merged**